node-repository/src/main/java/com/yahoo/vespa/hosted/provision/restapi/v2/NodesResponse.java

```diff
@@ -157,8 +157,11 @@ private void toSlime(Node node, boolean allFields, Cursor object) {
             toSlime(allocation.membership(), object.setObject("membership"));
             object.setLong("restartGeneration", allocation.restartGeneration().wanted());
             object.setLong("currentRestartGeneration", allocation.restartGeneration().current());
-            object.setString("wantedDockerImage", nodeRepository.dockerImage(node).withTag(allocation.membership().cluster().vespaVersion()).asString());
-            object.setString("wantedVespaVersion", allocation.membership().cluster().vespaVersion().toFullString());
+            String wantedVespaVersion = allocation.membership().cluster().vespaVersion().toFullString();
+            Optional<String> dockerImageRepo = allocation.membership().cluster().dockerImageRepo();
```

Comment:
Consider having a method on `ClusterSpec` that returns the full docker image

Reply:
Done